anaconda.spec.in

```diff
@@ -1,6 +1,6 @@
 # Avoid anaconda-core requiring gjs-console due to the GNOME welcome
 # screen that's shipped in it
-%global __requires_exclude_from ^%{_datadir}/anaconda/gnome/fedora-welcome.*$
+%global __requires_exclude_from ^%{_datadir}/anaconda/gnome/rhel-welcome.*$
```

Comment:
I think this fix should be part of the backport for https://github.com/rhinstaller/anaconda/pull/1742. However, couldn't we change this line to `%global __requires_exclude_from ^%{_datadir}/anaconda/gnome/*-welcome.*$` for both RHEL and upstream?

Reply:
You are right, maybe we even don't need it after anaconda-live sub-package was created
(commit ada65c38058870497239ea74b0db2c739c74f2e3 and some more) because the reason for original patch (commit 9152861b53db9f74d5f40db75d056c641c1d7aa9)  - dragging dependencies for anaconda-core - does not hold anymore and the proper solution has been implemented? 
from the commit:
"Ultimately perhaps the GNOME welcome screen should be split out from anaconda-core, but for now this will fix things up."
@M4rtinK what do you think?

Reply:
I think you're right, the dependency should wind up in the -live subpackage and it's valid there, and we don't have anything making anaconda-core depend on -live or anything. You could run a scratch build without the exclude line and run rpmdiff to confirm the difference, I guess.

Reply:
Opening upstream PR to remove the exclude:
https://github.com/rhinstaller/anaconda/pull/2529